### DISCUSSION TITLE
Why loki download logs in a reverse order only ?

### DISCUSSION THREAD

**Discussion by ahmedTouati** [grafana/grafana#46371] [General]
When downloading logs of a certain pod, the newest logs are on the top which is confusing for any user, is there any way to download in the normal order?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
